elastic/kibana
# [Index management] Add support in url for hidden toggle

## Summary

Gives the index list view in Index Management the ability to read a query param from the URL which will toggle the "include hidden indices" toggle above the table.

This work is preparatory for the linking functionality that will be added with Data streams.

### How to test

Currently there is no why in the UI to toggle this so it can be tested by:

1. Launching Kibana and going to Index Management
2. In the hash portion of the url append `?includeHidden=true` and press enter
3. The toggle should change

### Checklist
